gcc/rust/privacy/rust-visibility-resolver.cc

```diff
@@ -55,6 +56,7 @@ VisibilityResolver::resolve_visibility (const HIR::Visibility &visibility,
       return true;
     case HIR::Visibility::PUBLIC:
       // FIXME: We need to handle the restricted path here
+      // FIXME: We also need to handle 2015 vs 2018 edition conflicts
```

Comment:
I wouldn't worry about that untill the end of the milstone and we can think about it when we have something like libcore working and need to make decisions about versions.